Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -524,6 +532,9 @@ int main(int argc, char *argv[])
         return 0;
     }
 
+    if (headlessMode && useWPEPlatformAPI && !g_getenv("WPE_DISPLAY"))
+        wpeDisplay = wpe_display_headless_new();
+
```

Comment:
This should be protected by `ENABLE_WPE_PLATFORM_HEADLESS` too. I don't think we want to honor the env var when --headless is passed

Reply:
I went with the rationale that tweaking env vars would take precedence over command line options, but looks like other tools like `gcc` make cli switches (like `-L`) higher priority indeed.